docker-compose.yml

```diff
@@ -12,16 +12,16 @@ services:
     build:
       context: .
       target: devcontainer
-    devices:
-      - /dev/bus/usb:/dev/bus/usb
+#    devices:
+#      - /dev/bus/usb:/dev/bus/usb
       # - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
     volumes:
       - .:/workspace/frigate:cached
       - ./web/dist:/opt/frigate/web:cached
       - /etc/localtime:/etc/localtime:ro
       - ./config/config.yml:/config/config.yml:ro
       - ./debug:/media/frigate
-      - /dev/bus/usb:/dev/bus/usb
+#      - /dev/bus/usb:/dev/bus/usb
```

Comment:
Do we want to leave these out by default now?

Reply:
I'm not sure it matters for people developing on linux, it just complains on my mac. Might be worth considering but probably separate from this PR